avocado_vt/utils.py

```diff
@@ -195,12 +195,14 @@ def verify_background_errors(self):
         """
         err_file_path = os.path.join(self.logdir, BG_ERR_FILE)
         bg_errors = self.background_errors.get_all()
-        error_messages = ["BACKGROUND ERROR LIST:"]
+        error_messages = []
         for index, error in enumerate(bg_errors):
             error_messages.append(
                 "- ERROR #%d -\n%s" % (index, "".join(
                     traceback.format_exception(*error)
                     )))
+        if error_messages:
+            error_messages.insert(0, "BACKGROUND ERROR LIST:")
         genio.write_file(err_file_path, '\n'.join(error_messages))
```

Comment:
I think, write under `if error_messages`  is enough to avoid idle file.

Reply:
Good catch, will update it, thanks!

Reply:
@luckyh  Seems this part have not be updated yet ?